pkg/querier/parquet_queryable.go

```diff
@@ -499,6 +509,18 @@ func (q *parquetQuerierWithFallback) Select(ctx context.Context, sortSeries bool
 		sortSeries = true
 	}
 
+	queryIngesters := q.queryIngestersWithin == 0 || maxt >= util.TimeToMillis(time.Now().Add(-q.queryIngestersWithin).Add(-q.projectionHintsIngesterBuffer))
```

Comment:
This `maxt` is the original `maxt` or is the `maxt` already changed to not contain the timestamp for data in the ingesters?